### PR TITLE
Added calculation of launch angles of the RF trajectory from the source vertex.

### PR DESCRIPTION
**Pull request by FlahertyJustinC**
Implemented calculation and storage of launch angles of the RF signal from the source vertex to work in conjunction with [AraRoot Pull #68](https://github.com/ara-software/AraRoot/pull/68).  This stores the MCTruth values of the theta and phi launch angles in the Report pointer alongside the MCTruth values of the receiving angles of the signal.  The calculation of the launch angles follows the same framework as the receiving angles, which is a simple conversion of the Cartesian vector into theta and phi.

### CODE REVIEW

File: Report.cc [hunk 1]
```diff
@@ -4924,6 +4934,22 @@ void Report::GetAngleAnt(Vector &rec_vector, Position &antenna, double &ant_thet
 
 }
 
+void Report::GetAngleLaunch(Vector &launch_vector, double &launch_theta, double &launch_phi) {
+    
+    /*
+    2024-07-01 JCF
+    Takes the launch vector of the signal and calculates the theta and phi of the launch vector in the 
+    receiveing antenna's coordinate system.
```
Comment (clark2668) [inline Report.cc]: Does it make sense to calculate the _launch_ angle in the frame of the _target_? Wouldn't it make more sense to calculate the launch angle in the frame of the emitter? Or maybe that's what you actually do and the docs are just not quite right?

File: Report.cc [hunk 2]
```diff
@@ -4924,6 +4934,22 @@ void Report::GetAngleAnt(Vector &rec_vector, Position &antenna, double &ant_thet
...
+    
+    This is necessary because for source reconstruction, we need the launch vector of the RF signal in 
+    conjunction with the polarization to find the neutrino trajectory.
+    */
+    
+    launch_theta = launch_vector.Theta() * TMath::RadToDeg();
+    launch_phi = launch_vector.Phi() * TMath::RadToDeg();
+
+}
+
```
Comment (clark2668) [inline Report.cc:4953]: Some documentation about units and convention would be good -- it looks like this is reported in degrees? And with what convention, 0 = up, 180 = down, etc.

**Review by clark2668 — CHANGES_REQUESTED** [review]
Just a couple questions and requests for more clarifying docs.

File: Report.cc (hunk 2 above)
Comment (FlahertyJustinC) [reply Report.cc:4953]: Updated the description to have units and conventions for 0 and 180.

File: Report.cc (hunk 1 above)
Comment (FlahertyJustinC) [reply Report.cc]: Maybe my description was vague.  This calculates the launch angle theta and phi in station-centric coordinates, while launch angle in the Ray Tracer is calculated relative to the target vector.  I've updated the description of the function to hopefully make this more clear.

**Comment by FlahertyJustinC** [comment]
> Just a couple questions and requests for more clarifying docs.

Hey Brian.  Are my new pushes the changes you're looking for?  If so, could you approve them?  I can't merge until you've approved the changes.

**Review by clark2668 — APPROVED** [review]

**Comment by clark2668** [comment]
Whoops, sorry, looks okay to me.

**Pull request merged**